src/ast.c

```diff
@@ -606,7 +607,7 @@ static void ast_print_children(
 );
 
 /// Print a node.
-static void ast_print_node(
+void ast_print_node(
```

Comment:
To clarify, I’m suggesting renaming `ast_print_node()` to `ast_print_node_internal()` or whatever, and then creating the function above and exposing that instead (perhaps w/ an additional FILE* parameter, but since this is mostly for debugging stdout should do candidly...)

Reply:
Probably a very good idea

Reply:
Done as of acd725cd2ca7392e6871a70255c836c7135c3653